integration/airflow/openlineage/airflow/dag.py

```diff
@@ -120,7 +117,7 @@ def __init__(self, *args, **kwargs):
         kwargs["user_defined_macros"] = macros
         if kwargs.__contains__("lineage_custom_extractors"):
             for operator, extractor in kwargs['lineage_custom_extractors'].items():
-                extractor_mapper.add_extractor(operator, extractor)
+                extractor_manager.extractor_mapper.add_extractor(operator, extractor)
```

Comment:
I would encapsulate adding extractors within `ExtractorManager`:

```
extractor_manager.add_extractor()
```

That is, extractor map should be internal to and managed by `ExtractorManager`

Reply:
Well, maybe not. Looks like that's a pattern used throughout. I would have expected the `ExtractorManager` to define an interface for interacting with `extractors` and the underlying `extractor_mapper`

Reply:
Ok. Here I was just 'extracting' common functionality between three integrations, but I guess slight refactor won't hurt.

Reply:
Done.